README.md

```diff
@@ -35,6 +35,28 @@ Change all the details from one place: ``_data/data.yml``
 
 Watch my [video](https://www.youtube.com/embed/T2nx6tj-ZH4) on installation.
 
+## Preview locally using a Docker Image
+
+1. Build the image
+```sh
+docker build . -t online-cv --progress=plain
+```
+2. Use the image
+
+```sh
+docker container run -p 127.0.0.1:4000:4000 --name cv_container -d online-cv
```

Comment:
What do you think about add a share volume? That way you don't need to edit data inside the container.